docs/manual/actor.md

```diff
@@ -39,19 +39,19 @@ const federation = createFederation({
   // Omitted for brevity; see the related section for details.
 });
 
-federation.setActorDispatcher("/users/{handle}", async (ctx, handle) => {
-  // Work with the database to find the actor by the handle.
+federation.setActorDispatcher("/actors/{actorId}", async (ctx, actorId) => {
```

Comment:
There is one problem, the current URL template only allows `{handle}` as a placeholder. It would be better to rename the placeholder to `{identifier}` before the v1.0 release, I will work on it.